src/app/layout/header/menus.service.ts

```diff
@@ -111,4 +86,37 @@ export class MenusService {
     }
     return res;
   }
+
+  private getCreateMenuItems(): MenuItem {
+    const displayDeployments = (ENV === 'development');
+    let menus = [
+      {
+        name: 'Codebases',
+        path: ''
+      },
+      {
+        name: 'Pipelines',
+        path: 'pipelines'
+      },
+      {
+        name: 'Applications',
+        path: 'apps'
+      }
+    ];
+    if (displayDeployments) {
+      menus.push({
+        name: 'Deployments',
+        path: 'deployments'
+      });
+    }
+    menus.push({
+      name: 'Environments',
+      path: 'environments'
+    });
```

Comment:
Trivial nit:

Deployments will probably eventually replace both Applications and Environments and I feel like the order doesn't matter. Putting Deployments last will drop the need to have an extra push after the if statement.

Reply:
Yea, true. I'll touch that up.